_posts/2016/01/2016-01-29-elsewhere-on-the-web.md

```diff
@@ -0,0 +1,34 @@
+---
+layout: post
+authors: ["Greg Wilson"]
+title: "Elsewhere on the Web"
+date: 2016-01-29
+time: "06:00:00"
+category: ["Community"]
+---
+Our instructors teach a lot more than just Software Carpentry.
+For example,
+[Christie Bahlai]({{site.baseurl}}/team/#bahlai_christie) has started teaching a course on
+[Open Science and Reproducible Research](https://osrrcourse.wordpress.com/).
+The course blog already includes lots of interesting ideas,
+and there's lots more to come.
+
+Elsewhere,
+[Tiffany Timbers]({{site.basurl}}/team/#timbers_tiffany) is organizing several half-day workshops at Simon Fraser University
+as part of Titus Brown's
+[Data Intensive Biology training program]({{site.baseurl}}/blog/2016/01/online-workshops-from-ucdavis.html).
+These workshops will be hands on and supported by several TA's in addition to the instructor.
+All are welcome to participate, regardless of discipline or training level---please feel free to distribute these events widely.
+
```

Comment:
Might be good to add that if people cannot attend locally because of geography that they can watch the video stream on YouTube.